src/schemas/json/project.json

```diff
@@ -6,6 +6,7 @@
 
 	"definitions": {
 		"compilationOptions": {
+            "description": "Compilation options that are passed through to Roslyn.",
```

Comment:
Formatting is off here. Also, avoid the use of the term "Roslyn" since any compiler could be in use (F#, etc.). "Options that are passed to the compiler" is probably clearer.